packages/ui/src/modals/AddMembershipModal/AddMembershipSuccessModal.tsx

```diff
@@ -20,7 +21,7 @@ export function AddMembershipSuccessModal({ onClose, member }: Props) {
       <ModalBody>
         <Text>You have just successfully create a new membership</Text>
         <MemberRow>
-          <MemberInfo member={member} />
+          <MemberInfo member={(member as unknown) as MemberFieldsFragment} />
```

Comment:
Any way of getting rid of the cast here?

Reply:
I'd go fixing this when dealing with a general flow update – https://github.com/Joystream/pioneer/issues/166.

So ATM it's ugly, but should be rewritten as soon as we have that data fetched from query-node.